### PR TITLE
feat(cli): cdk version command

### LINKED ISSUE #4718: cdk version is not displaying installed version
<cdk --version> displays an old version
in my current scenario  it shows 1.13.1 (build 96cfc63),  even if I've updated to 1.15

### Reproduction Steps

I am working on  a Mac ( got the same behaviour on python 2.7.16 and also on python 3.7.4 )

### Environment

  - **CLI Version      :*aws --version
aws-cli/1.16.215 Python/2.7.16 Darwin/19.0.0 botocore/1.12.205*
  - **Framework Version:*it should be 1.15, but it shows 1.13*
  - **OS               :*Mac OS Catalina*
  - **Language         :*EN*

### Other

--- 

This is :bug: Bug Report

### PR DESCRIPTION
**Pull request by nmussy**
Alias `cdk --version` option with `cdk version` command

Fixes #4718

----

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

### CODE REVIEW

**Comment by shivlaks** [comment]
is there a basic test we can add here?

**Comment by nmussy** [comment]
@shivlaks I was going to try to reproduce from the `package.json` and `build-info.json`, but this looks simpler

File: packages/aws-cdk/test/integ/cli/test-cdk-version.sh [hunk 1]
```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+set -euo pipefail
+scriptdir=$(cd $(dirname $0) && pwd)
+source ${scriptdir}/common.bash
+# ----------------------------------------------------------
+
+setup
+
+assert "cdk version" "cdk --version"
```
Comment (eladb) [inline packages/aws-cdk/test/integ/cli/test-cdk-version.sh]: Did you ever run this test successfully? This is not the API for `assert`, which compares the result of the first command with STDIN (it doesn't take 2 arguments).

File: packages/aws-cdk/bin/cdk.ts [hunk 2]
```diff
@@ -230,6 +230,8 @@ async function initCommandLine() {
         } else {
           return await cliInit(args.TEMPLATE, language);
         }
+      case 'version':
+        return print(version.DISPLAY_VERSION);
```
Comment (eladb) [inline packages/aws-cdk/bin/cdk.ts]: `print` emits to STDERR while `--version` emits to STDOUT. You should use `data` instead.

File: packages/aws-cdk/test/integ/cli/test-cdk-version.sh [hunk 3]
```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
```
Comment (eladb) [inline packages/aws-cdk/test/integ/cli/test-cdk-version.sh]: Also, this file needs to be executable in order to run. It currently doesn't run.

**Pull request merged**